demos/signature/Makefile

```diff
@@ -0,0 +1,20 @@
+#
+# To run the demos when linked with a shared library (default):
+#
+#    LD_LIBRARY_PATH=../.. ./EVP_MD_demo
```

Comment:
EVP_MD_demo should be EVP_Signature_demo

Reply:
Done, thanks for your comments.